src/redux/toast/toast.sagas.js

```diff
@@ -0,0 +1,13 @@
+import { takeEvery } from 'redux-saga/effects';
+
+import { toast } from 'react-toastify';
+
+import { SET_TOAST_MESSAGE } from './toast.types';
+
+export function* handleSetToastMessage({ payload: message }) {
```

Comment:
We don't need saga for this.

Reply:
The hole logic is build around redux and redux-saga. It is done, to make reusability of toast easier